lib/public/components/RunDetail/index.js

```diff
@@ -122,19 +142,19 @@ const activeFields = (model) => ({
         name: 'Data Distribution (FLP)',
         visible: true,
         size: 'cell-s',
-        format: (boolean) => boolean ? 'On' : 'Off',
+        format: (boolean) => formatBoolean(boolean),
```

Comment:
simply give the function reference as property, no need to wrap in a function (same for the others)

Reply:
done!